src/networks.json

```diff
@@ -295,5 +295,15 @@
     "name": "kyve",
     "ownerAddress": "kyvevaloper1egqphd8yjdfv84fl825grwgna0pf2emagdmnz8",
     "gasPrice": "0.02ukyve"
+  },
+  {
+    "name": "quicksilver",
+    "ownerAddress": "quickvaloper1jdq7efmzmc8smscwmaw77vul7wptww74wskggw",
```

Comment:
I wasn't sure what exactly this mapped to but this is my valoper address, if it's just a pointer to the contributor to last update the network config and my address is appropriate.  let me know if I should change or omit it.

Reply:
ownerAddress is the validator it shows at the top of the validator list, it's for validators who run REStake UIs (or me for restake.app). I like the idea of contributor though! I'll remove through a PR suggestion now